packages/portalnetwork/src/wire/utp/PortalNetworkUtp/PortalNetworkUTP.ts

```diff
@@ -447,22 +447,8 @@ export class PortalNetworkUTP {
   }
   async handleResetPacket(request: HistoryNetworkContentRequest) {
     const requestCode = request.requestCode
+    await this.protocol.handleResetPacket(request.socket)
```

Comment:
Do we need to do this step at all since the next line deletes the socket from the table of open sockets?